students/Roy_Tate/lesson04/dict_lab.py

```diff
@@ -0,0 +1,23 @@
+
+user_dict = {'name':'chris', 'city':'Seattle', 'cake':'chocolate'}
+
+# Dictionaries 1 #
+
+print(user_dict)
+del user_dict['cake']
+print(user_dict)
+user_dict['fruit'] = 'Mango'
+print(user_dict)
+
+print(user_dict.keys())
+print('\nCake in user_dict? ' + str(('cake' in user_dict.keys())))
```

Comment:
The `in` operator checks for key membership in dict, so this can be `'cake' in user_dict`